external/kafka/src/main/scala/org/apache/spark/streaming/kafka/KafkaUtils.scala

```diff
@@ -684,3 +679,53 @@ private class KafkaUtilsPythonHelper {
     kafkaRDD.offsetRanges.toSeq
   }
 }
+
+private object KafkaUtilsPythonHelper {
+  private var initialized = false
+
+  def initialize(): Unit = {
```

Comment:
Since `initialize()` is already called in `object KafkaUtilsPythonHelper`, the purpose of adding the `initialize` method is only for loading the class of `object KafkaUtilsPythonHelper`, right?

If changing it to
```
object KafkaUtilsPythonHelper {

  SerDeUtil.initialize()
  new PythonMessageAndMetadataPickler().register()

...
  def createPicklerIterator: Iterator[Array[Byte]] = {
    new SerDeUtil.AutoBatchedPickler(iter)
  }
...
}
```
When calling KafkaUtilsPythonHelper.createPicklerIterator in the closure, `SerDeUtil.initialize()` and `new PythonMessageAndMetadataPickler().register()` will be called when loading class of `object KafkaUtilsPythonHelper`.